fastlane/helpers/VLC.rb

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/ruby
+
+class VLC
+
+  class InfoPlistPath
+    IOS = '../Sources/VLC for iOS-Info.plist'
```

Comment:
For iOS we also need to update the watchkit and the watchkit extension with the same number :)